src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutesSpec.scala

```diff
@@ -307,7 +307,9 @@ class GoogleExtensionRoutesSpec extends GoogleExtensionRoutesSpecHelper with Sca
     // A duplicate call should return OK and an empty response
     Post(s"/api/google/resource/${resourceType.name}/foo/${resourceType.ownerRoleName.value}/sync") ~> routes.route ~> check {
       status shouldEqual StatusCodes.OK
-      responseAs[Map[WorkbenchEmail, Seq[SyncReportItem]]] shouldBe empty
+      val response = responseAs[Map[WorkbenchEmail, Seq[SyncReportItem]]]
+      response.keys should not be empty
```

Comment:
(Ditto for the existing assertion on the initial sync call.)

Can we make this assertion more meaningful, rather than only checking emptiness?  E.g. is there something we know about the expected `WorkbenchEmail` that we can assert?

Reply:
Asserted that the response contains an email starting with `policy-`. The part after `policy-` could be anything, but is likely a UUID, but I didn't think that was too important to check. The API should just be expecting _an_ email, not a specific email, since there's no way for the caller to know the email before the call!